### DISCUSSION TITLE
Base URL and assets

### DISCUSSION THREAD

**Discussion by IslamHeza** [ng-matero/ng-matero#571] [Q&A]
Hello all,

**First**, Thank you @nzbin for your effort in this dashboard.

I need to understand how to load the **assets** folder including languages. In my app when I let **base URL** empty in the **environment** file the assets load fine from localhost ( http://localhost:4200/assets/i18n/en-US.json).

However, when I put my backed server domain as base URL it fails and "Problem with 'en-US' language initialization" appears as image attached. (http://backend-server/api/assets/i18n/en-US.json).

I don't need to load the assets from the backend, so how to handle this please.

Thank you.

![image](https://github.com/ng-matero/ng-matero/assets/30021771/bbb94693-7b2c-4ffd-b35c-086eed26ed7d)

**Comment by nzbin** [comment]
I'm not sure about your problem and I can't find any issues with the latest version. Maybe you should tell me the version you use and the baseUrl you define.

You can also remove the following codes if you don't need i18n.
https://github.com/ng-matero/ng-matero/blob/2a5759f277e04c451dfb5292e00b5a2ede813436/src/app/app.module.ts#L25-L27
